WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -267,6 +270,7 @@ public void showMagicLinkSentScreen(String email) {
     @Override
     public void openEmailClient() {
         if (WPActivityUtils.isEmailClientAvailable(this)) {
+            AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_MAGIC_LINK_OPEN_EMAIL_CLIENT_VIEWED);
```

Comment:
This stat should actually be tracking when the `LoginMagicLinkSentFragment` is viewed, rather than when the user attempts to open the email client.  I realize the naming makes that a bit unclear.